src/ipmanager/ui/views.py

```diff
@@ -1,8 +1,16 @@
 # Create your views here.
 from django.views import View
-from django.http import HttpResponse
 from django.shortcuts import render
+from django.views.generic import ListView
+from ipmanager.api.models import Group
 
 class HomeView(View):
   def get(self, request):
     return render(request, "ui/index.html", {})
+
+class GroupListView(ListView):
+ model = Group
+ template_name = 'ui/groupListView.html'
```

Comment:
Python convention is to use "snake case" for template names (like variables and functions), so this would be `ui/group_list.html`